docs/balances/FAQs.md

```diff
@@ -0,0 +1,21 @@
+---
+tags: [Balances]
+---
+
+# Balance FAQs
+
+## How does balances differ from valuations?
+
+Valuations provide a history of the value of a portfolio whereas balances provide a near realtime current value of a portfolio.
+
+## Why are cash and holdings balances not returned with total balance?
+
+This is to avoid large responses from portfolios that hold many positions.
+
+## Why does this portfolio not have any balances?
+
+Balances are currently lazily created upon initial funding. If the portfolio has no transactions then the portfolio's balances will not exist.
```

Comment:
We'll need to change this as soon as we release https://wealthkernel.atlassian.net/browse/WK2-155